index.html

```diff
@@ -1,15 +1,47 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@300&family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css" />
     <title>Robert Kanyoro</title>
-</head>
-<body>
-    <header class="top_Section">
-
-    </header>
-</body>
-</html>
\ No newline at end of file
+  </head>
+  <body>
+    <main class="top_Section">
+      <section class="header">
+        <div class="app_bar">
+          <p>9:41</p>
+          <ul class="icons_phone">
+            <li><img src="icons/Cellular Connection.png" alt="Network Cell"></li>
+            <li><img src="icons/Wifi.png" alt="wifi"></li>
+            <li><img src="icons/Battery.png" alt="Battery"></li>
+          </ul>
+        </div>        
```

Comment:
- [ ] Please I highly recommend removing the div tag with `app_bar` class which is not part of the required figma design. 😃 

_**REQUIRED FIGMA DESIGN**_

![Screenshot (238)](https://user-images.githubusercontent.com/78104544/235059273-510d5437-9493-441c-83e6-e559e5ce1eb0.png)


_**YOUR DESIGN**_

![Screenshot (237)](https://user-images.githubusercontent.com/78104544/235059320-9b62483d-ea38-48ec-bde7-0bbc25b623dd.jpg)